app/models/Restaurant.java

```diff
@@ -11,10 +13,11 @@
  */
 @Entity
 @Table(name = "abh_restaurant")
-public class Restaurant extends Model{
+public class Restaurant extends Model implements Validation{
     @Id
     @Column(name = "restaurantId", columnDefinition = "BIGINT")
     @JsonView(RestaurantViews.BasicDetails.class)
```

Comment:
Now that is very logical and I have no clue why I thought that I'm supposed to create view class for each model. Also, I will make sure not to use plural class names again.